### PR TITLE
not all sagas can be correlated by >1 attribute

### PR DESCRIPTION
**Pull request by mauroservienti**
Makes clear that is the saga in this specific sample that is correlated by more than 1 attribute. I read the original sentence:

> A saga can be correlated with messages using more than one property

as: sagas in general can be correlated by multiple properties. But this is not the case. What I'm trying to make clear is that it's the saga in this specific sample that needs to be correlated by multiple properties.

### CODE REVIEW

**Comment by bording** [comment]
@mauroservienti Looks like someone else touched these files. Conflicts!

File: samples/saga/sagafinder-process.include.md [hunk 1]
```diff
@@ -1 +1 @@
-It is important to note that the saga is not sending the order ID to the payment processor. Instead, it is sending a payment transaction ID. A saga can be correlated with messages using more than one property. For example, `OrderId` and `PaymentTransactionId`. This requires both of these properties to be treated as unique.
\ No newline at end of file
+It is important to note that the saga is not sending the order ID to the payment processor. Instead, it is sending a payment transaction ID. This saga needs to be correlated by more than one attribute. For example, `OrderId` and `PaymentTransactionId`. This requires both of these properties to be treated as unique.
```
Comment (bording) [inline samples/saga/sagafinder-process.include.md]: There is a mix of "attribute" and "property" here. What term do we use for these things in the rest of the saga documentation? We should be consistent.

Comment (mauroservienti) [reply samples/saga/sagafinder-process.include.md]: property is the correct one. Fixed.

**Comment by udidahan** [comment]
I haven't looked at the code but this sounds like a case where the saga does simple request/response with another endpoint, is that correct? If so, NServiceBus handles the correlation automatically not requiring any saga finding logic.

**Pull request merged**